TESTING.md

```diff
@@ -13,3 +13,25 @@ In a new terminal window start the tests using:
 ```sh
 yarn e2e:test
 ```
+
+## Running end to end tests against Sauce Labs
+
+```sh
+yarn e2e:build && yarn e2e:serve
```

Comment:
can we combine these into one command?

Reply:
My recommendation: document how to do this with the docker command, so we can run tests as they are run in CI.